showcase/scripts/cleanup_otelcol.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Directories to delete
+for ((i = 1; i <= 10; i++)); do
+    directory="../directory$i"
```

Comment:
Given that we are probably adding in more metrics in the future (which also adds in more test cases to showcase), I don't think we should be hard coding the number of directories. I think in a comment above, it would be better if this could all be done in a single directory with different files, but there may be an issue with doing that (I'm not entirely sure, so let me know otherwise).

If directories are needed and the other option isn't possible, I think it would be better if the directories could be named (rather than having something like 1,2,3).

Reply:
Yes, working on making single directory possible.

Reply:
Done.